libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -304,8 +304,17 @@ uint16_t SRV_Channel::get_limit_pwm(Limit limit) const
 // return true if function is for a multicopter motor
```

Comment:
We should change the existing comment a bit to clarify that this returns true for more than just "multicopter motors".